src/control/cmd/ddb/ddb_types.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+type CommandContext struct {
+	CContext          *DdbContext
```

Comment:
CContext name is a little confusing, since that's not a C type. I know the C context is embedded in there, but maybe something like `ddbCtx` would be clearer?